bootstrap/master.sh

```diff
@@ -25,13 +25,26 @@ fi
 systemctl daemon-reload
 systemctl restart kubelet
 
-echo "Inititializing the master..."
+# execute modprobe on node - workaround for heketi gluster
```

Comment:
Somehow this module doesn't get loaded into system by default. In Heketi project they do this manually in a Ansible playbook. module is apparently part of LMV and needed for heketi